types/ckeditor__ckeditor5-core/src/editor/editorconfig.d.ts

```diff
@@ -88,14 +89,16 @@ export interface EditorConfig {
     mention?: MentionConfig;
     pagination?: PaginationConfig;
     placeholder?: string;
-    plugins?: Array<string | typeof Plugin>;
+    plugins?: Array<string | (new (editor: Editor) => Plugin)>;
```

Comment:
2C: this one should be probably exported as reusable type, describing a plugin class

Reply:
I'll use `PluginInterface` that is exported from `core/plugin`.